packages/bpk-docs/src/pages/InfiniteScrollPage/InfiniteScrollPage.js

```diff
@@ -0,0 +1,173 @@
+/*
+ * Backpack - Skyscanner's Design Systrem
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import withInfiniteScroll, {
+  onItemsFetch,
+} from 'bpk-component-infinite-scroll';
+import BpkButton from 'bpk-component-button';
+import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
+import { cssModules } from 'bpk-react-utils';
+
+import infiniteScrollReadme from '../../../../bpk-component-infinite-scroll/readme.md';
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import IntroBlurb from './../../components/neo/IntroBlurb';
+
+import STYLES from './InfiniteScrollPage.scss';
+
+const getClassName = cssModules(STYLES);
+
+const elementsArray = [
+  'Item 1',
+  'Item 2',
+  'Item 3',
+  'Item 4',
+  'Item 5',
+  'Item 6',
+  'Item 7',
+  'Item 8',
+  'Item 9',
+  'Item 10',
+  'Item 11',
+  'Item 12',
+  'Item 13',
+  'Item 14',
+  'Item 15',
+  'Item 16',
+  'Item 17',
+  'Item 18',
+  'Item 19',
+  'Item 20',
+];
+
+const List = ({ elements }) => (
+  <div className={getClassName('bpk-docs-infinite-scroll-page__list')}>
+    {elements.map(element => <div key={element}>{element}</div>)}
+  </div>
+);
+
+List.propTypes = {
+  elements: PropTypes.arrayOf(PropTypes.any).isRequired,
+};
+
+const FixedHeightPanel = ({ children }) => (
+  <div className={getClassName('bpk-docs-infinite-scroll-page__fixed-panel')}>
+    {children}
+  </div>
+);
+
+FixedHeightPanel.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const InfiniteList = withInfiniteScroll(List);
+
+const components = [
+  {
+    id: 'infinite',
+    title: 'Infinite list',
+    blurb: [
+      'The infinite scroll component allows long lists to be progressively loaded as you scroll',
```

Comment:
This needs to move up to the page level as its an intro for the whole page, not just this configuration

Reply:
but there's one there already.
```
    blurb={
      isNeo
        ? []
        : [
            <IntroBlurb>
              This High-order component is used to wrap a list component and add
              the functionality to allow that list to load more elements as the
              user scrolls.
            </IntroBlurb>,
          ]
    }
```
I left the first empty the same as other components do.